arbeitszeit/use_cases/show_my_plans.py

```diff
@@ -65,6 +67,7 @@ def show_company_plans(self, request: ShowMyPlansRequest) -> ShowMyPlansResponse
             for plan, cooperation in all_plans_of_company
             if (
                 not plan.is_approved
+                and not plan.is_rejected
```

Comment:
no, in the UI non-active plans are those that are not rejected nor approved. So I guess the two conditions 
                and not plan.is_active_as_of(now)
                and not plan.is_expired_as_of(now)
probably would have been enough from the start since a plan that is not active cannot be approved at the same time. Does that make sense to you?